### PR TITLE
feat: implement reset wallet flow

### PR DESCRIPTION
**Pull request by gabaldon**
close #312

### CODE REVIEW

File: lib/screens/login/view/login_screen.dart [hunk 1]
```diff
@@ -203,6 +205,10 @@ class LoginScreenState extends State<LoginScreen>
     ];
   }
 
+  bool showBiometrics() {
+    return (!globals.testingActive && Platform.isIOS) || Platform.isAndroid;
```
Comment (Tommytrg) [inline lib/screens/login/view/login_screen.dart:209]: We should avoid polluting the code with test logic. Is there any other way to know if the biometrics are enabled? If not, I'd move this to an external function such as `isBiometricsEnabled` which handles that logic.

File: lib/shared/api_database.dart [hunk 2]
```diff
@@ -215,7 +215,6 @@ class ApiDatabase {
           'fileExists': fileExists
         },
       );
-
```
Comment (Tommytrg) [inline lib/shared/api_database.dart:218]: Could we add this empty line?

File: lib/main.dart [hunk 3]
```diff
@@ -19,8 +20,8 @@ void main() async {
   if (Platform.isWindows || Platform.isMacOS || Platform.isLinux) {
     await windowManager.ensureInitialized();
     WindowOptions windowOptions = WindowOptions(
-      size: Size(400, 700),
-      minimumSize: Size(400, 700),
+      size: Size(430, 730),
```
Comment (parodyBit) [inline lib/main.dart:23]: changing this results in a different aspect ratio.

File: lib/screens/login/view/login_screen.dart (hunk 1 above)
Comment (parodyBit) [inline lib/screens/login/view/login_screen.dart:209]: We should disable biometrics when running e2e tests and later add a unit test for the biometrics widget.
`Platform.isIOS` and `Platform.isAndroid` can have support biometrics or not depending if the app is being ran on a physical device or in an emulator.

maybe update this bit to ->
```
bool showBiometrics() {
    return !globals.testingActive;
```

File: lib/widgets/PaddedButton.dart [hunk 4]
```diff
@@ -140,12 +143,13 @@ class PaddedButton extends StatelessWidget {
           ? theme.textButtonTheme.style?.copyWith(
               foregroundColor: MaterialStateProperty.all(color),
               overlayColor:
-                  MaterialStateProperty.all(Color.fromARGB(16, 255, 255, 255)))
+                  MaterialStateProperty.all(Color.fromARGB(16, 126, 126, 126)))
```
Comment (parodyBit) [inline lib/widgets/PaddedButton.dart:146]: should we add this to `lib/theme/colors.dart` ?

File: lib/screens/login/view/login_screen.dart (hunk 1 above)
Comment (parodyBit) [reply lib/screens/login/view/login_screen.dart:209]: yeah, we check for that in [lib/screens/login/bloc/login_bloc.dart#L39](https://github.com/witnet/my-wit-wallet/blob/main/lib/screens/login/bloc/login_bloc.dart#L39)
i think there is an issue / edge case if the host device supports biometrics and is running the e2e tests via an emulator.

File: integration_test/e2e_re_establish_wallet.dart [hunk 5]
```diff
@@ -0,0 +1,54 @@
+import 'dart:io';
```
[the remark this answers is not in the document]
Comment (gabaldon) [reply integration_test/e2e_re_establish_wallet.dart:1]: It is needed for checking the platform

File: lib/screens/login/view/login_screen.dart (hunk 1 above)
Comment (gabaldon) [reply lib/screens/login/view/login_screen.dart:209]: Biometrics can only be shown in IOS and Android, we still need that check. It does not matter if IOS is running on a physical device or an emulator, e2e testing does not detect the native popups. Is Android only detecting native pop ups on physical device?

Comment (gabaldon) [reply lib/screens/login/view/login_screen.dart:209]: We don't need to check if biometrics are enabled, we need to check if there is a testing environment to disable it, otherwise it will show the native popup and the e2e testing will break in iOS.

**Review by Tommytrg — APPROVED** [review]

**Review by Tommytrg — APPROVED** [review]
